.tee-cloud/compose-files/tee-compose.yaml

```diff
@@ -0,0 +1,12 @@
+version: "3"
```

Comment:
hmm i think we should move this within the agent-tee-phala/ folder as its related to it 

and maybe we move the current contents of agent-tee-phala/ to a subfolder, 'image'

e.g.:

agent-tee-phala/
  .tee-cloud/ 
  image/
    .env.example
    Dockerfile
    ...